docs/03-AcceptanceTests.md

```diff
@@ -310,13 +319,65 @@ $I->seeInCurrentUrl('user/1');
 $user_id = $I->grabFromCurrentUrl('~$/user/(\d+)/~');
 ```
 
-## Selenium WebDriver
+## WebDriver: Selenium Server or PhantomJS
 
 A nice feature of Codeception is that most scenarios are similar, no matter of how they are executed.
 `PhpBrowser` was emulating browser requests but how to execute such test in a real browser like Chrome or Firefox? 
 Selenium WebDriver can drive them so in our acceptance tests we can automate scenarios we used to test manually.
 In such tests we should concentrate more on **testing the UI** than on testing functionality.
 
+"[Selenium WebDriver](http://www.seleniumhq.org/projects/webdriver/)" is the name of an API (specified by the Selenium project)
+to drive browsers automatically. Codeception supports two implementations of this API: Selenium Standalone Server and PhantomJS
+
+### Selenium Standalone Server
+
+Selenium Server is a piece of software that "drives" (i.e. runs) your local browser by sending commands (i.e. your test cases) to it.
+
+Overview of components:
+```
+Codeception -> facebook/php-webdriver -> Selenium Server -> Gecko/Chrome Driver -> Browser (Firefox or Chrome)
+```
+
+Installation instructions:
+
+1. You need [Firefox](http://www.mozilla.com/) and/or [Chrome](https://www.google.com/chrome/browser/desktop/) :-)
+1. You need [Java](http://www.java.com/)
+1. Download [Selenium Standalone Server](http://www.seleniumhq.org/download/)
+1. Download [geckodriver](https://github.com/mozilla/geckodriver/releases) (for Firefox) and/or
+[ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) and extract the executable to a folder where Selenium Server can find it
+(best guess: just drop it in the same folder where `selenium-server-standalone-xxx.jar` lives)
```

Comment:
On Windows it works if it's in the same directory. Not so in Linux?  
(Idea: No reason to create a directory for a single file, which is **only** used by Selenium anyway)

WebDriver: Yeah, but I only noticed that afterwards ;-) However, I'm going to rewrite (i.e. mostly delete) this section in the Module.  
Reason: Installation is really a headache!!! I spent hours (if not days) to get all this shit (Selenium etc.) up and running. This very page would probably be the only resource on the entire web to explain **everything** (not just some ridiculous part). And here is the better place than in the Module, cause: If people can't get the entire package up and running in 15 minutes, they're frustrated and never going to look into specific modules. At that stage, they don't even know what "module" means....